docs/multicluster/antctl.md

```diff
@@ -1,11 +1,16 @@
 # Antctl Multi-cluster commands
 
 Starting from version 1.6.0, Antrea supports the `antctl mc` commands, which can
-collect information from a leader cluster in a ClusterSet, for troubleshooting
-issues in an Antrea Multi-cluster ClusterSet.
+collect information from a leader cluster in a ClusterSet for troubleshooting
+issues in an Antrea Multi-cluster ClusterSet, create and delete resources in an
+Antrea Multi-cluster ClusterSet, and so on. The command `antctl mc get` is supported
+since Antrea version 1.6.0 and other commands are supported from 1.7.0. These commands
+cannot run inside the `antrea-controller`, `antrea-agent` and the `antrea-mc-controller`
```

Comment:
Remove "the" before `antrea-mc-controller`.

Reply:
Done. Removed.